src/Layout/Presentation/DistrictOverview/DistrictOverview.tsx

```diff
@@ -24,20 +24,20 @@ export class DistrictOverview extends React.Component<DistrictOverviewProps, {}>
         const highestVsAverageInPercentage = (1 / highestVotingPower / (1 / averageVotingPower)) * 100;
         const lowestVsAverageInPercentage = (1 / lowestVotingPower / (1 / averageVotingPower)) * 100;
         const mostWeightedDistrict = (
-            <span className="has-text-success">
+            <span className="is-size-4">
```

Comment:
I propose we simply show normal text, having some words in a larger font feels a bit strange to me. This applies to the other cases where we change from coloured to large text as well.

Reply:
Done